src/libraries/System.Private.CoreLib/src/System/Reflection/RuntimeMethodBody.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Reflection
+{
+    internal sealed partial class RuntimeMethodBody : MethodBody
+    {
+        private byte[] _IL = null!;
```

Comment:
These `= null!` initializers should be in the fake CoreCLR constructor of this type. They are unnecessary for Mono.